applications/nrf_desktop/doc/ble_bond.rst

```diff
@@ -40,9 +40,9 @@ Both IDs related to the given Bluetooth peer operation are propagated in ``ble_p
 
 The identity usage depends on the device type:
 
-* nRF Desktop central
+nRF Desktop central:
   The nRF Desktop central uses only one application local identity and only one Bluetooth local identity (the default ones).
-* nRF Desktop peripheral
+nRF Desktop peripheral:
```

Comment:
Why remove list here? This will blend all this lines into one paragraph. Please revert the edit.

Reply:
Yes, can do. I believe the issue is that the formatting does not come out correctly:
![c1](https://user-images.githubusercontent.com/51916561/145172448-a1d6db00-4051-49a3-876b-8a0ab31f27f4.PNG)
But in this case I can just remove the repeated device names in the each bullet.



Otherwise, this is how my suggested change would look:
![c2](https://user-images.githubusercontent.com/51916561/145172620-7db61c84-f5cc-4f6c-a63f-05d4a0f3aa44.PNG)
![c1](https://user-images.githubusercontent.com/51916561/145172626-058ad954-7d53-413d-8492-cfb21d886afd.PNG)

Reply:
@peknis @wiba-nordic There is this one that I have no changed yet, what do you both think?

Reply:
Oh yes. Your suggestion looks good. I like the term list formatting.

Reply:
Agreed. Though make sure that everything that should be part of the latter definition actually becomes part of it in the final build preview, and everything that should doesn't. There were several indented paragraphs there, after all.

Reply:
Okay then that is the second image (right?), and then I will leave it as is, because that was my original suggestion.

Reply:
Okay, will do.